scenario:
  name: 'Scenario Absolute and Relative'

  overview:  "Sample scenario which does two orbit changes and do a drifting manoeuvre to reach a specific relative
              position with respect to another spacecraft."

  prop_type: 2-body

  CheckPoints:
    S0:
      id: 0
      position:
        kep:
          a: 7084.12176561 - 20.0
          e: 7084.12176561 * 0.0008099 / (7084.12176561 - 47.0)
          i: 1.71925752363
          w: 0.533974758001
          O: 0.396507390127
          v: 0.0

    S1:
      id: 1
      position:
        kep:
          a: 7084.12176561 - 16.0
          e: 7084.12176561 * 0.0008099 / (7084.12176561 - 16.0)
          i: 1.71925752363
          w: 0.533974758001
          O: 0.396507390127
          v: 0.0

    S2:
      id: 2
      position:
        lvlh:
          R: [-4.0, -10.0, 0.0]
      error_ellipsoid: [0.4, 1.5, 0.4]
      manoeuvre: drift
